app/src/main/java/com/github/sdpteam15/polyevents/EventActivity.kt

```diff
@@ -60,7 +79,53 @@ class EventActivity : AppCompatActivity() {
         findViewById<ImageView>(R.id.img_event_logo).apply {
             //TODO : change image
         }
+
+        if (event.isLimitedEvent()) {
+            subscribeButton.visibility = View.VISIBLE
+            if (currentDatabase.currentUser != null
```

Comment:
Shouldn't we test that the event attendee limit is not exceeded before displaying the button ?

Reply:
Well my thought was if we hide the button if the event is full, we might give the impression that the event is a public event. I think it's better to keep the button so that when the user presses, he gets shown that the event is limited but full.

Reply:
So the button is visible only if the event is a limited event, regardless if it's full or not.

Reply:
Ok, with the explanation, it's fine for me !